src/java.desktop/share/classes/javax/swing/JFormattedTextField.java

```diff
@@ -1086,7 +1086,8 @@ protected void setEditValid(boolean valid) {
         }
 
         /**
-         * Subclass and override if you wish to provide a
+         * @implNote Default implementation returns null.
```

Comment:
I suggest moving `@implNote` to the end of the spec, otherwise the entire spec is the implementation note, which is wrong, isn't it?

```suggestion
         * @implNote The default implementation returns <code>null</code>.
```
It requires the definite article.

Both points above also apply to `getNavigationFilter` method below.